clojurians_log/playbooks/roles/geerlingguy.memcached/.travis.yml

```diff
@@ -0,0 +1,26 @@
+---
+services: docker
+
+env:
+  - distro: centos7
+  - distro: ubuntu1604
+  - distro: ubuntu1404
+  - distro: ubuntu1204
+
+script:
+  # Configure test script so we can run extra tests after playbook is run.
+  - export container_id=$(date +%s)
+  - export cleanup=false
+
+  # Download test shim.
+  - wget -O ${PWD}/tests/test.sh https://gist.githubusercontent.com/geerlingguy/73ef1e5ee45d8694570f334be385e181/raw/
```

Comment:
Might want to check the sha1sum of this downloaded file before running it.

Reply:
This is from a vendored dependency. Don't think @bombaywalla has written it, was there as it is before.